src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.DecisionDagRewriter.cs

```diff
@@ -160,6 +186,51 @@ public override BoundNode VisitCompoundAssignmentOperator(BoundCompoundAssignmen
                     return null;
                 }
 
+                public override BoundNode VisitConversion(BoundConversion node)
```

Comment:
>VisitConversion [](start = 42, length = 15)

Consider capturing somewhere that function pointers will likely need special handling in this walker as well.